bin/deferred_work_manager

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require "tom_queue"
+
+module TomQueue
+  class Daemon
+    include LoggingHelper
+
+    def run(argv = ARGV)
+      return unless load_rails!(".")
+
+      deferred_manager = DeferredWorkManager.new
+      deferred_manager.start
+    end
+
+    # Load Rails environment
+    #
+    # Returns true/false indicate whether Rails is loaded or not
+    def load_rails!(path)
+      rails_path = File.expand_path(File.join(path, 'config/environment.rb'))
+      if File.exists?(rails_path)
+        info "Loading rails from `#{rails_path}`"
+
+        ENV['RACK_ENV'] ||= ENV['RAILS_ENV'] ||= 'development'
+        require rails_path
+        ::Rails.application.eager_load!
+        true
+      else
+        false
+      end
```

Comment:
https://github.com/fac/tom_queue/pull/22#discussion_r45495606 Oh, I thought we just did what we discussed on Slack. That's not quite progressive TMH. Anyway will open another branch _just_ fix the signal handling. Thanks for the reviewing @caius @joshado :+1:

Close the pull request for now.